src/integrationTest/java/uk/gov/hmcts/reform/pcqbackend/service/PcqDisposerServiceIntegrationTest.java

```diff
@@ -0,0 +1,105 @@
+package uk.gov.hmcts.reform.pcqbackend.service;
+
+import net.serenitybdd.annotations.WithTag;
+import net.serenitybdd.annotations.WithTags;
+import org.junit.Rule;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.system.OutputCaptureRule;
+import org.springframework.test.util.ReflectionTestUtils;
+import uk.gov.hmcts.reform.pcqbackend.domain.ProtectedCharacteristics;
+import uk.gov.hmcts.reform.pcqbackend.repository.ProtectedCharacteristicsRepository;
+import uk.gov.hmcts.reform.pcqbackend.util.PcqIntegrationTest;
+
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@WithTags({@WithTag("testType:Integration")})
+public class PcqDisposerServiceIntegrationTest extends PcqIntegrationTest {
+
+    public static final String CASE_ID = "9dd003e0-8e63-42d2-ac1e-d2be4bf956d9";
+
+    @Value("${disposer.keep-with-case:92}")
```

Comment:
You already defined the default values in application.yaml

Reply:
Integration tests fail without this set here.